src/ray/object_manager/object_manager.cc

```diff
@@ -902,24 +905,13 @@ std::string ObjectManager::DebugString() const {
 }
 
 void ObjectManager::RecordMetrics() const {
-  int64_t used_memory = 0;
-  for (const auto &it : local_objects_) {
-    object_manager::protocol::ObjectInfoT object_info = it.second.object_info;
-    used_memory += object_info.data_size + object_info.metadata_size;
-  }
-  stats::ObjectManagerStats().Record(used_memory,
-                                     {{stats::ValueTypeKey, "used_object_store_memory"}});
-  stats::ObjectManagerStats().Record(local_objects_.size(),
-                                     {{stats::ValueTypeKey, "num_local_objects"}});
-  stats::ObjectManagerStats().Record(active_wait_requests_.size(),
-                                     {{stats::ValueTypeKey, "num_active_wait_requests"}});
-  stats::ObjectManagerStats().Record(
-      unfulfilled_push_requests_.size(),
-      {{stats::ValueTypeKey, "num_unfulfilled_push_requests"}});
-  stats::ObjectManagerStats().Record(pull_requests_.size(),
-                                     {{stats::ValueTypeKey, "num_pull_requests"}});
-  stats::ObjectManagerStats().Record(profile_events_.size(),
-                                     {{stats::ValueTypeKey, "num_profile_events"}});
+  stats::ObjectStoreAvailableMemory().Record(config_.object_store_memory - used_memory_);
```

Comment:
Why don't we keep this 

```
for (const auto &it : local_objects_) {	  stats::ObjectStoreUsedMemory().Record(used_memory_);
    object_manager::protocol::ObjectInfoT object_info = it.second.object_info;	  stats::ObjectStoreLocalObjects().Record(local_objects_.size());
    used_memory += object_info.data_size + object_info.metadata_size;	  stats::ObjectManagerWaitRequests().Record(active_wait_requests_.size());
  }
```

and do RAY_CHECK(used_memory_ == used_memory) here for testing (since we cannot have any test now)? I think we can remove it in a week or right before 1.0 branch cut or something.